tlv493d/tlv493d.go

```diff
@@ -422,6 +425,18 @@ func (d *Dev) StopContinousRead() {
 	d.continuousReadWG.Wait()
 }
 
+// CalibrateTemperatureOffsetCompensation computes the temperature offset compensation based upon a reading from the sensor compared to the actual temperature.
+// The returned value must be stored and passed in Opts.TemperatureOffsetCompensation in future uses of TLV493D driver.
+// See TLV493D user manual, "3.2 Calculation of the temperature" for more information
```

Comment:
Generally prefer to end sentences with a period.

Reply:
Fixed